src/main/java/lotto/constants/LottoConfiguration.java

```diff
@@ -0,0 +1,30 @@
+package lotto.constants;
+
+public enum LottoConfiguration {
+    MIN_PRICE(1000),
+    MAX_PRICE(100000),
```

Comment:
저와 똑같이 현행법상 구매 제한을 걸어두셨군요!! ㅋㅋㅋㅋㅋㅋ
위와 같이 숫자의 자리수가 많아지면 `_`를 통해 자리수를 구분시켜 줄 수 있습니다!
```suggestion
    MAX_PRICE(100_000),
```

Reply:
현행법...이게 바로 로또를 사는 사람이라 지켜지더라구요ㅋㅋㅋㅋ
_ 에 대해서 하나 더 배워 갑니다.
사소하지만 잊혀지기 쉬운 부분인거 같아요.
감사합니다.